ui/src/components/Sidebar/Sidebar.tsx

```diff
@@ -20,6 +20,8 @@ type SideBarProps = {
   setFilters: React.Dispatch<React.SetStateAction<LogFilters>>;
   drawerOpen: boolean;
   setDrawerOpen: Function;
+  setValidFromTimestamp: Function;
+  setValidUntilTimestamp: Function;
```

Comment:
The three type definitions here that are `Function` should instead be `React.Dispatch<React.SetStateAction<T>>`, using the `setFilters` type defintion as an example